tests/functional/spec/interstitial_definition.spec.js

```diff
@@ -8,36 +8,20 @@ describe("Component: Interstitial Definition", () => {
 
     it("When there is a definition on an interstitial, then the page is displayed correctly", () => {
       expect($(InterstitialDefinitionPage.definitionTitle(1)).isDisplayed()).to.be.true;
-      expect($(InterstitialDefinitionPage.definitionContent(1)).isDisplayed()).to.be.false;
-      expect($(InterstitialDefinitionPage.definitionButton(1)).isDisplayed()).to.be.false;
+      expect($(InterstitialDefinitionPage.definitionContent(1)).getText()).to.equal("");
 
       expect($(InterstitialDefinitionPage.definitionTitle(2)).isDisplayed()).to.be.true;
-      expect($(InterstitialDefinitionPage.definitionContent(2)).isDisplayed()).to.be.false;
-      expect($(InterstitialDefinitionPage.definitionButton(2)).isDisplayed()).to.be.false;
+      expect($(InterstitialDefinitionPage.definitionContent(2)).getText()).to.equal("");
```

Comment:
This one needs to check for existence of something else than empty string and the tests below need similar fix.

Reply:
Same as the one above